effect/src/test/scala/shims/effect/TaskInstancesSpecs.scala

```diff
@@ -38,9 +41,20 @@ import org.typelevel.discipline.specs2.Discipline
 
 object TaskInstancesSpecs extends Specification with Discipline {
   import TaskArbitrary._
+  import Task.taskParallelApplicativeInstance
 
-  def is =
-    br ^ checkAllAsync("Task", implicit ctx => EffectTests[Task].effect[Int, Int, Int])
+  def is = taskEff ^ taskPar ^ parTaskApp
```

Comment:
Do we need `br` between each of these?

Reply:
Note the line you deleted. :-) It just reformats the output. For the record, this file is a particularly odd use of specs2.

Reply:
Whoops, completely missed that :+1: